tools/summonerd/src/participant.rs

```diff
@@ -11,14 +11,14 @@ use penumbra_proto::{
         participate_response::Confirm, ParticipateRequest, ParticipateResponse,
     },
 };
-use tokio::sync::mpsc;
+use tokio::sync::{mpsc, Mutex};
 use tonic::{Status, Streaming};
 
 use crate::phase::Phase;
 
 pub struct Participant {
     address: Address,
-    rx: Streaming<pb::ParticipateRequest>,
+    rx: Mutex<Streaming<pb::ParticipateRequest>>,
```

Comment:
I don't think we should be using a `Mutex` here, why are we sharing access to the participant's messages?